pydeseq2/dds.py

```diff
@@ -58,18 +62,22 @@ class DeseqDataSet(ad.AnnData):
         DataFrame containing sample metadata.
         Must be indexed by sample barcodes.
 
-    design_factors : str or list
-        Name of the columns of metadata to be used as design variables.
-        (default: ``'condition'``).
+    design : str or pandas.DataFrame
```

Comment:
not related to this PR, but since you are using typehints anyway, you could rely on [sphinx-autodoc-typehints](https://pypi.org/project/sphinx-autodoc-typehints/) and not have to repeat the type information in the docstrings.

Reply:
Good suggestion, I created #329 to keep track of this.